config/locales/curation_concerns.en.yml

```diff
@@ -101,17 +102,29 @@ en:
         property: keywords
       title:
         property: name
+
```

Comment:
Nope, it was in models, and at least some of it is necessary. It's confusing to me what parts in the stack are using it, but I figure investigating that is separate from trying to merge the projects.